spec/features/org_admin_can_inactive_recipient_with_funds_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.feature "Org admin can inactive recipient with funds" do
+  scenario "Org admin changes a recipient with fundings to inactive" do
+    organization = Organization.create!(name:"Homes for Humanity", description: "We build homes", status: 1)
+    user = organization.users.create(username: 'fiona@cat.com', password: 'password')
+    role = Role.create(name: 'org_admin')
+    user.roles << role
+    country = Country.create(name:"Ghana")
+    recipient = organization.recipients.create(name:"Bob", description:"Loves building huts", country_id: country.id, image_path: "https://robohash.org/Bob", amount_received: 20)
+
+    visit login_path
+    fill_in 'Username', with: 'fiona@cat.com'
+    fill_in 'Password', with: 'password'
+    click_button 'Login'
+
+    click_link "My Organization"
+    click_link "Manage Recipients"
+
```

Comment:
To make this test more robust in the future, I'd recommend putting the deactivate and reactivate buttons in a div specific to the recipient so it's clear.